app/api/predictions/service.py

```diff
@@ -78,29 +81,67 @@ async def get_new_prediction(cls, prompt: str, client_websocket):
     @classmethod
     async def get_daily_event(cls):
         try:
-            current_day = datetime.now().strftime('%A').upper()
-            response = cls().events.query(
-                IndexName='day-index',
-                KeyConditionExpression=boto3.dynamodb.conditions.Key('day').eq(current_day)
+            
+            # THIS SHOULD BE TE CODE, THE OTHER PART IS HARDCODED SO WE CAN GET AN EVENT
+            # Get the current date and time
+            # current_time = datetime.now()
+            # current_timestamp = int(current_time.timestamp())  # Convert to UNIX timestamp
+            
+            # # Query the DynamoDB table with date range filter
+            # response = cls().events.scan(
+            #     FilterExpression=boto3.dynamodb.conditions.Attr('start_ts').lte(current_timestamp) &
+            #                      boto3.dynamodb.conditions.Attr('end_ts').gte(current_timestamp)
+            # )
+            
+            iso_date_str = '2024-08-17T20:30:00+0000'
```

Comment:
Since we are going to predict events, we probably we to have as a param, instead of hard-coded date.